$NetBSD: patch-aa,v 1.9 2009/12/25 17:50:52 heinz Exp $

  Do not hardcode share/man as the subdirectory for man pages.
  Submitted upstream as
    http://sourceforge.net/tracker/?func=detail&aid=2026068&group_id=3978&atid=103978

  manifypods-razor depends on INST_MAN5DIR which gets created by the 'config'
  target. Without this, there may problems building with parallel make.
  Submitted upstream as
    http://sourceforge.net/tracker/?func=detail&aid=2921151&group_id=3978&atid=103978

--- Makefile.PL.orig	2007-05-09 00:47:53.000000000 +0200
+++ Makefile.PL
@@ -54,13 +54,14 @@ sub constants {
     my $man5 = sprintf <<'MAKE', $prefix;
 
 # begin razor-agents
+MANSUBDIR = share/man
 MAN5PODS = docs/razor-agent.conf.pod \
 	   docs/razor-whitelist.pod  \
 	   docs/razor-agents.pod
 INST_MAN5DIR = blib/man5
-INSTALLMAN5DIR = $(DESTDIR)$(PREFIX)/share/man/man5
-INSTALLSITEMAN5DIR = $(DESTDIR)$(SITEPREFIX)/share/man/man5
-INSTALLVENDORMAN5DIR = $(DESTDIR)$(VENDORPREFIX)/share/man/man5
+INSTALLMAN5DIR = $(DESTDIR)$(PREFIX)/$(MANSUBDIR)/man5
+INSTALLSITEMAN5DIR = $(DESTDIR)$(SITEPREFIX)/$(MANSUBDIR)/man5
+INSTALLVENDORMAN5DIR = $(DESTDIR)$(VENDORPREFIX)/$(MANSUBDIR)/man5
 MAN5EXT = 5
 # end razor-agents
 
@@ -115,7 +116,7 @@ sub MY::manifypods {
 # begin razor-agents
 %s
 
-manifypods-razor : docs/razor-agent.conf.pod \
+manifypods-razor : config docs/razor-agent.conf.pod \
 	docs/razor-agents.pod                \
 	docs/razor-whitelist.pod
 	$(POD2MAN) \
